modules/core/shared/src/test/scala/gem/arb/ArbTime.scala

```diff
@@ -78,12 +77,7 @@ trait ArbTime {
     }
 
   implicit val arbDuration: Arbitrary[Duration] =
-    Arbitrary {
-      for {
-        a <- Gen.choose(-10000L, 10000L)
-        u <- Gen.oneOf(ChronoUnit.values.filterNot(_.isDurationEstimated))
-      } yield Duration.of(a, u)
-    }
+    Arbitrary(Gen.posNum[Long].map(Duration.ofMillis))
```

Comment:
Why this change?

Reply:
We had two conflicting definitions of `Arbitrary[Duration]` before.  The one I changed it to was in `gem.config.Arbitraries` but really belonged in `ArbTime`.  Like `Arbitrary[Timestamp]`, we have to limit the possible values to those that will work when written to the database (say in an `exposure_time` column). In this case, that's duration values that can be mapped to/from milliseconds:

```scala
  implicit val DurationMeta: Meta[Duration] =
    Distinct.long("milliseconds").timap(Duration.ofMillis)(_.toMillis)
```

I actually don't see a constraint on positive exposure times but it seems like there should be one!  We could probably also keep the definition that was there but constrain it so that `ChronoUnit` is at least `MILLIS`.

I suppose another idea would be to use an explicit `Duration` and `Timestamp` generator for database tests.